src/components/Slider/Slider.module.css

```diff
@@ -0,0 +1,110 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # SLIDER
+\*------------------------------------*/
+
+/**
+ * Slider wrapping Label and Input
+ */
+.slider {
+  display: flex;
+  flex-direction: column;
+  gap: var(--eds-size-2-and-half);
+}
+
+/**
+ * Slider Input
+ */
+.slider__input {
+  --slider-track-height: var(--eds-size-1);
+  --slider-thumb-size: var(--eds-size-2-and-half);
+
+  appearance: none;
+  background: transparent;
+
+  height: var(--slider-thumb-size);
+}
+.slider__input:focus {
+  outline: none;
+}
+
+/**
+ * Slider Input Track
+ */
+/* Firefox */
+.slider__input::-moz-range-track {
+  /* fills left side of track as a percentage of the input value */
+  background: linear-gradient(
```

Comment:
Would be nice to use `accent-color` but to my knowledge only colors the filled track and is [too modern](https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color#browser_compatibility) for [our browser policy](https://czi.atlassian.net/wiki/spaces/LP/pages/2057733851/Edu+Tech+Inclusive+Browser+Support+Policy)